packages/airnode-node/src/api/index.ts

```diff
@@ -23,10 +23,9 @@ function buildOptions(payload: CallApiPayload): adapter.BuildRequestOptions {
 
   switch (aggregatedApiCall.type) {
```

Comment:
The `http-signed-data-gateway` section and `http-gateway` section look the same, can you merge them?